### DISCUSSION TITLE
Profile (WLAN) für DTU anlegen, möglich?

### DISCUSSION THREAD

**Discussion by H83v** [tbnobody/OpenDTU#1420] [General]
Hi, 

ich hätte mal zu folgenden Thema eine Frage:

Ist-Zustand: ich habe eine OpenDTU für mein HMS 800 und lass mir darüber die Daten meine PV Anlage anzeigen.
Jetzt bekommen meine Eltern auch den gleichen Wechselrichter an ihr BKW. Diese wohnen aber wo anders und 
haben dementsprechend ein andere WLAN Netzwerk.

Gibt es hier die Möglichkeit mit der OpenDTU 2 Profile für die beiden Standorte anzulegen, damit ich die DTU in Einzelfällen mal bei meinen Eltern zum anzeigen der Daten verwenden kann?

Oder gibt es sogar die Möglichkeit, dass ich bei mir zu Hause an die Daten meiner Eltern zu kommen, ohne die DTU umstecken zu müssen?

VG Hannes

**Comment by Tiese** [comment]
Hallo Hannes,

am Einfachsten wird es sein, dass du dir dein **Profil** - die Config-Datei sicherst, dann die Daten deiner Eltern eingibst und dieses **Profil** ebenfalls sicherst - jeweils in einem extra Ordner.
Dann kannst du das jeweils benötigte einfach laden!

Und ein Zugang über VPN sollte auch aus der Ferne möglich sein (habe ich aber noch nie getestet).

Schöne Grüße
Tiese

**Reply by H83v** [reply]
Hi Tiese,

das klingt nach einer sehr guten Idee. Werd das mal so umsetzen.

Wenn es noch andere Ansätze gibt (z.B. Lösung aus der Ferne a la VPN), bin ich offen.

Grüße Hannes

**Reply by MichlW** [reply]
VPN oder irgendein anderer Remote-Zugriff kann nicht funktionieren, wenn deine DTU nicht am entsprechenden Standort ist. Die DTU funkt ja lokal auf einer bestimmten Frequenz direkt die Wechselrichter an.

Natürlich kannst du über VPN eine dortige DTU erreichen, aber das war ja nicht deine beschriebene Ausgangssituation.

**Reply by Tiese** [reply]
@MichlW: das funktioniert sehr wohl - gerade getestet! Es sei denn, mein WLAN reicht ca. 7 km?! EDIT: Natürlich muss jeweils ein DTU vorhanden sein!

@H83v = Hannes: Wenn deine Eltern eine Fritz!Box haben und du ein iPhone (das waren meine Testbedingungen) kannst du so vorgehen...
- Mit dem iPhone ins WLAN deiner Eltern einloggen
- die App: MyFRITZ! herunterladen
- dann einfach das machen, was die App vorschlägt
- zuletzt musst du noch die VPN-Verbindung (wird nach korrekter Einrichtung unter Einstellungen angezeigt) im iPhone aktivieren

Schon kannst du außerhalb des WLAN-Netzwerks deiner Eltern unter der ganz normalen internen IP (192.168.178.xxx) auf openDTU zugreifen - übrigens auch auf z.B. das SmartMeter (Tasmota) :-)

Schöne Grüße
Tiese

**Reply by MichlW** [reply]
@Tiese Genauso war es gemeint. Ich meinte natürlich nicht, dass VPN nicht funktionieren kann, sondern dass seine eine vorhandene DTU die Wechselrichter bei den Eltern über VPN nicht erreichen wird.

**Reply by Tiese** [reply]
@MichlW: 👍

**Reply by H83v** [reply]
Hi ihr Beiden,

ok, dann ist die OpenDTU bei meinen Eltern Grundvoraussetzung. Klingt plausibel und dann kann ich per VPN diese aus der Ferne  ansprechen, wie von @Tiese beschrieben. 

Danke euch
